src/Datadog.Trace.ClrProfiler.Native/environment_variables.h

```diff
@@ -67,6 +67,17 @@ const WSTRING log_path = "DD_TRACE_LOG_PATH"_W;
 // https://github.com/dotnet/coreclr/issues/12468
 const WSTRING clr_disable_optimizations = "DD_CLR_DISABLE_OPTIMIZATIONS"_W;
 
+// Sets whether to intercept method calls when the caller metohd is inside a
+// domain-neutral assembly. This is dangerous because the integration assembly
+// Datadog.Trace.ClrProfiler.Managed.dll must also be loaded domain-neutral,
+// otherwise a FileNotFound exception will occur. This setting should only be
+// enabled when there is only one AppDomain or, when hosting applications in IIS,
+// the user can guarantee that all Application Pools on the system only have one
+// application.
+// Default is false.
```

Comment:
Nice comment!

Reply:
"at most one application" per Application Pool (zero is valid!)

Reply:
Inserted "at most one application" in 1a79b7b5239ab20af134e26484cca6db56e29cb5